### ISSUE TITLE
Shield is disabled at the beginning of the next round

### ISSUE THREAD

**Issue by Rencos1** [krysztalzg/ThandulsTogglableEffects#6]
Shield should be disabled at the beginning of you next turn, not at the beginning of the next round.
A little change can fix this issue in effects.js
Combat utility belt will not update the effect on the token if a duration is not set when the effect is created. Must be greater or equal to 0.2 to be sure that turnStart is triggerered first.
```javascript
static shield() { 
        return {
            name: "Shield",
            label: "Toggled Effect: Shield",
            icon: "modules/ThandulsTogglableBuffsAndEffects/media/shield.jpg",
            duration: getDurationData(1), 
            flags: {
                dae: {
                    specialDuration: ["turnStart"]
                }
            },
            changes: [
                {key: "data.attributes.ac.value", mode: 2, value: "5"},
              ],
        };
    }
```

**Comment by kszczepanskidev** [comment]
My initial plan was to not use DAE module, but base effects have way to many restrictions as you can see. I'll be updating all connected effects to DAE module assist if needed

**Comment by kszczepanskidev** [comment]
This should be fixed now in [1.2.5](https://github.com/krysztalzg/ThandulsTogglableEffects/releases/tag/1.2.5).
